kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -221,6 +260,16 @@ define([
 
                 // CELL messages
                 case RESPONSES.RUN_STATUS:
+                    if (msgData.event === 'launched_job_batch') {
```

Comment:
extract job ID mapping information from the `launched_job` and `launched_job_batch` run status messages